package.json

```diff
@@ -60,5 +60,10 @@
 			"unicorn/prefer-string-slice": "off",
 			"unicorn/prefer-includes": "off"
 		}
+	},
+	"tsd": {
+		"compilerOptions": {
+			"isolatedModules": true
```

Comment:
by adding this, it fails without the change in this PR

![image](https://user-images.githubusercontent.com/1404810/77361798-3c051a00-6d50-11ea-94de-fba6ad026ef1.png)